src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/NoHardcodedEnvironmentUrlsRuleTests.cs

```diff
@@ -186,7 +190,7 @@ public void DisallowedHostsMatchingTest(string testString, bool isMatch)
         [DataRow("http://subdomain1.management.azure.com", false)]
         [DataRow("https://subdomain1.management.azure.com", false)]
         [DataRow("all the world is a stage, but subdomain1.management.azure.com should not be hardcoded", false)]
-        [DataRow("all the world is a stage, but subdomain1.schema.management.azure.com should not be hardcoded", true)]
+        [DataRow("all the world is a stage, but subdomain1.1schema.management.azure.com should not be hardcoded", false)]
```

Comment:
Why did this change?

Reply:
It's an exact repeat of a line in the 'true' section (note the true arg).  Something like this was probably the intention.